index.js

```diff
@@ -1,39 +1,99 @@
 async function setupPlugin({config, global}) {
-    global.posthogUrl = config.postHogUrl
+    global.posthogUrl = cleanInstanceUrl(config.postHogUrl)
     global.apiToken = config.postHogApiToken
     global.projectToken = config.postHogProjectToken
-    global.syncScoresIntoPosthog = global.posthogUrl && global.apiToken && global.projectToken
+    global.zapierUrl = config.zapierUrl
+    global.actionIdToName = {}
+    config.actionIdToName.split(',').forEach(pair => {
+        const [actionId, conversionName] = pair.split(':')
+        global.actionIdToName[parseInt(actionId)] = conversionName
+    })
+    global.defaultStartTime = config.defaultStartTime ? new Date(config.defaultStartTime) : new Date(2021, 6, 1)
+    global.isConfigSet = global.posthogUrl
+        && global.apiToken
+        && global.projectToken
+        && global.zapierUrl
+        && Object.keys(global.actionIdToName).length
+        && isValidDate(global.defaultStartTime)
 }
 
+function cleanInstanceUrl(url) {
+    return url.replace(/\/$/, '')
```

Comment:
What are we doing here? Clearing the trailing slash? 

- We can make this name more descriptive if that's the case – but I'd suggest not making this a function and just calling `url.replace(/\/$/, '')` in the one place we need to do it (setupPlugin)